netjsonconfig/backends/openwrt/openwrt.py

```diff
@@ -88,6 +88,15 @@ def wireguard_auto_client(cls, **kwargs):
                 }
             ],
         }
+        if data['client']['ip_address']:
+            config['interfaces'][0]['addresses'] = [
+                {
+                    'proto': 'static',
+                    'family': 'ipv4',
```

Comment:
We can't know whether the IP address will belong to IPv4 or IPv6 family since we are dealing with variables here.